awe_languagetool/languagetoolClient.py

```diff
@@ -56,31 +56,36 @@ class languagetoolClient:
     def __init__(self, port=8081):
 
         self.port = port
-        self.MAPPING_PATH = \
-            resources.path('awe_languagetool',
-                           'languagetool_rulemapping.json')
-        print(self.MAPPING_PATH)
-
+        path_context = resources.as_file(
+            resources.files('awe_languagetool').joinpath('languagetool_rulemapping.json')
+        )
+        with path_context as out_path:
+            self.MAPPING_PATH = str(out_path)
```

Comment:
as of python3.11, resources.path() is deprecated. importlib's docs say that the equivalent replacement is "as_file(files(package).joinpath(resource))".

the original code uses the path strictly as a str, so that's what the cast is for.

Reply:
Comment this in the code.